app/models/job/JobService.scala

```diff
@@ -72,11 +73,11 @@ class JobService @Inject()(wkConf: WkConf,
       organization <- organizationDAO.findOne(user._organization)(GlobalAccessContext)
       resultLink = jobAfterChange.resultLinkPublic(organization.name, wkConf.Http.uri)
       resultLinkSlack = jobAfterChange.resultLinkSlackFormatted(organization.name, wkConf.Http.uri)
+      workflowLink = jobAfterChange.workflowLinkSlackFormatted(wkConf.Http.uri)
       multiUser <- multiUserDAO.findOne(user._multiUser)(GlobalAccessContext)
       superUserLabel = if (multiUser.isSuperUser) " (for superuser)" else ""
       durationLabel = jobAfterChange.duration.map(d => s" after ${formatDuration(d)}").getOrElse("")
-      msg = s"Job ${jobBeforeChange._id} succeeded$durationLabel. Command ${jobBeforeChange.command}, organization: ${organization.displayName}.${resultLinkSlack
-        .getOrElse("")}"
+      msg = s"Job ${jobBeforeChange._id} succeeded$durationLabel. Command ${jobBeforeChange.command}, organization: ${organization.displayName}.$resultLinkSlack$workflowLink"
```

Comment:
Is it always one or the other?  otherwise I would at least add a space inbetween `$resultLinkSlack$workflowLink`

Reply:
Both include their own leading space if they are set. This way, no double spaces are included if both are none.